README.rst

```diff
@@ -22,16 +22,56 @@ python-can
 
 The **C**\ ontroller **A**\ rea **N**\ etwork is a bus standard designed
 to allow microcontrollers and devices to communicate with each other. It
-has priority based bus arbitration, reliable deterministic
+has priority based bus arbitration and reliable deterministic
 communication. It is used in cars, trucks, boats, wheelchairs and more.
 
 The ``can`` package provides controller area network support for
-Python developers; providing `common abstractions to
-different hardware devices`, and a suite of utilities for sending and receiving
+Python developers; providing common abstractions to
+different hardware devices, and a suite of utilities for sending and receiving
 messages on a can bus.
 
 The library supports Python 2.7, Python 3.4+ as well as PyPy 2 & 3 and runs on Mac, Linux and Windows.
 
+
+Features
+--------
+
+- common abstractions for CAN communication
+- support for many different backends [**TODO** some list?]
```

Comment:
Or link to the latest release docs - https://python-can.readthedocs.io/en/master/interfaces.html